packages/account/src/Sections/Security/Passkeys/passkeys.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { PlatformContext, routes } from '@deriv/shared';
+import { Redirect } from 'react-router-dom';
+import { useIsPasskeySupported } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+
+const Passkeys = observer(() => {
+    const { is_passkeys_enabled } = React.useContext(PlatformContext);
+    const { is_passkey_supported, is_loading } = useIsPasskeySupported();
+    const { ui } = useStore();
+    const { is_mobile } = ui;
```

Comment:
Maybe we can consider useBreakpoint from quill-design ? 
Since BreakpointProvider is wrapped in app.jsx in core level, it can be used everywhere. Just a suggestion since this removes store usage completely in this file

Reply:
will proceed with quill hook for this new component